v1.1/null-handling.md

```diff
@@ -262,7 +262,7 @@ For example, let's say you want to calculate the average value of column `b` as
 
 ## NULLs and Sorting
 
-When sorting a column containing `NULL` values, CockroachDB orders `NULL`s lower than the first non-`NULL` value. This differs from PostgreSQL, which orders `NULL`s higher than the last non-`NULL` value.
+When [sorting a column](select.html#sorting-retrieved-values) containing `NULL` values, CockroachDB orders `NULL`s lower than the first non-`NULL` value. This differs from PostgreSQL, which orders `NULL`s higher than the last non-`NULL` value.
```

Comment:
Perhaps add the `NULL`s first with `ASC`, and `NULL`s last  with `DESC` here, too?

I guess the example below shows that pretty clearly but `NULL`s lower than the first non-`NULL` isn't the case with `DESC`.

Reply:
Done. PTAL.